cluster-scope/overlays/moc/zero/secret-generator.yaml

```diff
@@ -5,5 +5,5 @@ metadata:
   name: secret-generator
 files:
   - groups/cluster-admins.enc.yaml
-  - oauths/moc-sso-client-secret.enc.yaml
+  - oauths/apicurio.enc.yaml
```

Comment:
Please revert this change, instead list the encrypted resource along other user groups in the `moc/common` overlay as described above.

Reply:
done